src/components/FeatureChecking/withFeatureProvider.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from 'react';
+import FeatureProvider from './FeatureProvider';
+import type { FeatureConfig } from './flowTypes';
+
+type WrapperProps = {
+    features: FeatureConfig,
+};
+
+function withFeatureProvider(WrappedComponent: React.ComponentType<any>) {
```

Comment:
Can you add a test, maybe snapshot?

Reply:
https://github.com/box/box-ui-elements/pull/751/files#diff-198ef77bc61f0032026472fb03a0210b